doc/rados/operations/monitoring.rst

```diff
@@ -781,10 +781,20 @@ the Mean Time Between Failures (MTBF) and Mean Time To Recover (MTTR)
 for each pool. The availability score is then calculated by finding 
 the ratio of MTBF to the total time.  
 
-The score is updated every five seconds. This interval is currently 
-not configurable. Any intermittent changes to the pools that 
-occur between this duration but are reset before we recheck the pool 
-status will not be captured by this feature. 
+The score is updated every one second. Any intermittent changes to 
+the pools that occur between this duration but are reset before we 
```

Comment:
Transient changes to pools that occur and are reverted between successive updates will not be captured.